routing/routing_helpers.hpp

```diff
@@ -34,6 +34,10 @@ void ReconstructRoute(IDirectionsEngine & engine, RoadGraphBase const & graph,
                       my::Cancellable const & cancellable, bool hasAltitude,
                       vector<Junction> & path, Route & route);
 
+/// \brief Converts |edge| to |segment|.
+/// \returns false if mwm of |edge| is not alive.
+void EdgeToSegment(NumMwmIds const & numMwmIds, Edge const & edge, Segment & segment);
```

Comment:
I'll need this method in PedestrianDirectionsEngine as well to convert Edges to Segments. I'll use it in next PR.